docs/source/development/commands_reference.md

```diff
@@ -373,14 +372,6 @@ kedro build-docs
 The `build-docs` command builds [project documentation](../tutorial/package_a_project.md#add-documentation-to-your-project) using the [Sphinx](https://www.sphinx-doc.org) framework. To further customise your documentation, please refer to `docs/source/conf.py` and the [Sphinx documentation](http://www.sphinx-doc.org/en/master/usage/configuration.html).
 
 
-#### Lint your project
-
-```bash
-kedro lint
-```
-
-Your project is linted with [`black`](https://github.com/psf/black), [`flake8`](https://gitlab.com/pycqa/flake8) and [`isort`](https://github.com/PyCQA/isort).
```

Comment:
https://github.com/kedro-org/kedro/pull/1912#issuecomment-1271287376
@MerelTheisenQB wrt your comment in #1912, should we keep this in for now?

Reply:
As discussed, removed the changes in this file(`docs/source/development/commands_reference.md`) from this PR as we are not removing mentions of `kedro lint` and `kedro test` from the docs yet. The deprecation warnings for `kedro test` and `kedro lint` in docs will be addressed in #1912